Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN curl -L -o maven.tar.gz https://dlcdn.apache.org/maven/maven-3/3.8.8/binarie
 RUN ./maven/bin/mvn --version && \
 	./maven/bin/mvn clean package
 
-FROM registry.ci.openshift.org/origin/4.16:jenkins
+FROM registry.redhat.io/ocp-tools-4/jenkins-rhel8:v4.14.0
```

Comment:
Why are we using 4.14 image?

Reply:
The "4.16" version in OpenShift CI is basically broken and old/obsolete. This uses a version of Jenkins that we released.